app/presenters/publishing_api_presenters/publication_helper.rb

```diff
@@ -0,0 +1,15 @@
+module PublishingApiPresenters::PublicationHelper
```

Comment:
I think this name might lead to confusion with Publications. Perhaps just move it into the `Edition` base presenter instead?